usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -40,16 +40,16 @@
     "  transaction_normalized.description AS transaction_description,",
     "  transaction_normalized.modification_number,",
     "",
-    "  COALESCE(transaction_fpds.place_of_perform_country_n, transaction_fabs.place_of_perform_country_n) AS pop_country_name,",
-    "  COALESCE(transaction_fpds.place_of_perform_country_c, transaction_fabs.place_of_perform_country_c) AS pop_country_code,",
+    "  COALESCE(transaction_fpds.place_of_perform_country_n, transaction_fabs.place_of_perform_country_n, 'USA') AS pop_country_name,",
```

Comment:
Is this right?  Do we want to update name in addition to the code?

Reply:
Good catch; that was not intended behavior